.devcontainer/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/devcontainers/python:3-bookworm
+
+RUN export DEBIAN_FRONTEND=noninteractive \
+     && apt-get update && apt-get install -y xdg-utils \
+     && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+
+ENV POETRY_HOME="/opt/poetry"
+ENV PATH="$POETRY_HOME/bin:$PATH"
+RUN curl -sSL https://install.python-poetry.org | python3 -
```

We don't really want to depend on the script here, and we're not actively encourage it's use either. Instead a manual install flow is perfectly suitable, in my opinion:

```suggestion
ENV POETRY_HOME=/opt/poetry
RUN python3 -m venv $POETRY_HOME && \
    $POETRY_HOME/bin/pip install poetry

ENV VIRTUAL_ENV=/opt/poetry-env
RUN python3 -m venv $VIRTUAL_ENV  
    
ENV PATH="$VIRTUAL_ENV/bin:$POETRY_HOME/bin:$PATH"
```

This way we have a virtualenv holding Poetry (like the script creates, but explicitly controlled), and a separate virtual env for the project. This is a generic template following our recommendations that can be duplicated by other projects making use of Poetry + Dev Containers as well.